src/generator/grammar.ne

```diff
@@ -26,6 +26,7 @@ const lexer = moo.compile({
   label: ":",
   integer: /[0-9]+/,
   id: {match: /[a-zA-Z_]+/, type: moo.keywords({timer: 'timer', reset: 'reset'})},
+  string: /[a-zA-Z][a-zA-Z_0-9]*/,
```

Comment:
What do you think about renaming "string" to something like "alphanum" or "varname"?

Reply:
Seems good to me